build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    kotlin("jvm") version "1.4.0"
+    id("org.jmailen.kotlinter") version "3.0.2"
+    id("com.github.johnrengelman.shadow") version "6.0.0"
```

Comment:
produces "fat jars" that have dependencies bundled in. This is a sort of regrettable way to package software, but it's probably the simplest for your users.